speechmatics/config.py

```diff
@@ -0,0 +1,70 @@
+import copy
+import os.path
+import logging
+import yaml
+
+LOGGER = logging.getLogger(__name__)
+DEFAULT_CONFIG_PATH = os.path.expanduser("~/.config/speechmatics/config.yaml")
+
+
+class ConfigurationFormatError(ValueError):
+    pass
+
+
+def load_config(path=None):
+    """
+    Loads configuration yaml file into python dict object
+
+    Args:
+        path (Union[str, None]): File path to a config to load. Can be None, in
+            that case DEFAULT_CONFIG_PATH is used.
+
+    Returns:
+        dict: Config representation as a dict.
+
+    Raises:
+        FileNotFoundError: If configuration file does not exists. Note, that if
+            no `path` is passed and DEFAULT_CONFIG_PATH does not exist, this
+            function silently returns empty dict.
+        ConfigurationFormatError: If configuration file has unexpected format.
+    """
+    orig_path = path
+    if path is None:
+        path = DEFAULT_CONFIG_PATH
+
+    try:
+        with open(path, 'r') as config_file:
+            config = yaml.safe_load(config_file)
```

Comment:
I wasn't aware of this function method and how it differs from the standard `load`.
Nice, TIL!